lib/portus/background/registry.rb

```diff
@@ -14,9 +14,13 @@ def sleep_value
       # options and because the `execute!` method will deal with valid rows
       # already.
       def work?
-        true
+			  true
       end
 
+			def enabled?
+			  return APP_CONFIG("background.registry")
```

Comment:
Don't use `return`, ruby will implicitly return the last eval'd object. Moreover, it should be like this:

```
APP_CONFIG.enabled?("background.registry")
```